core/src/main/scala/tech/beshu/ror/configuration/SslConfiguration.scala

```diff
@@ -83,13 +83,16 @@ sealed trait SslConfiguration {
   def keystorePassword: Option[SslConfiguration.KeystorePassword]
   def keyPass: Option[SslConfiguration.KeyPass]
   def keyAlias: Option[SslConfiguration.KeyAlias]
+  def truststoreFile: Option[JFile]
```

Comment:
maybe we can create wrapper for JFile for Truststore and Keystore to avoid confusions?